src/cascadia/LocalTests_SettingsModel/SerializationTests.cpp

```diff
@@ -0,0 +1,304 @@
+                                                            "name": "BhojwaniShell"
+                                                        },
+                                                        {
+                                                            "antialiasingMode": "aliased",
+                                                            "name": "NiksaShell"
+                                                        }
+                                                    ]
+                                                },
+                                                "schemes": [
+                                                    {
+                                                        "name": "Cinnamon Roll",
+
+                                                        "cursorColor": "#FFFFFD",
+                                                        "selectionBackground": "#FFFFFF",
+
+                                                        "background": "#3C0315",
+                                                        "foreground": "#FFFFFD",
+
+                                                        "black": "#282A2E",
+                                                        "blue": "#0170C5",
+                                                        "cyan": "#3F8D83",
+                                                        "green": "#76AB23",
+                                                        "purple": "#7D498F",
+                                                        "red": "#BD0940",
+                                                        "white": "#FFFFFD",
+                                                        "yellow": "#E0DE48",
+                                                        "brightBlack": "#676E7A",
+                                                        "brightBlue": "#5C98C5",
+                                                        "brightCyan": "#8ABEB7",
+                                                        "brightGreen": "#B5D680",
+                                                        "brightPurple": "#AC79BB",
+                                                        "brightRed": "#BD6D85",
+                                                        "brightWhite": "#FFFFFD",
+                                                        "brightYellow": "#FFFD76"
+                                                    }
+                                                ],
+                                                "actions": [
+                                                    {"command": { "action": "renameTab","input": "Liang Tab" },"keys": "ctrl+t" }
+                                                ],
+                                                "keybindings": [
+                                                    { "command": { "action": "sendInput","input": "VT Griese Mode" },"keys": "ctrl+k" }
+                                                ]

```

Comment:
Wait how the heck does this pass if the keybindings aren't getting re-serialized?

Reply:
Ah. We actually copy the Json::Value for actions/keybindings. Then when we serialize our settings, we put them back in.